client/src/unifyfs.c

```diff
@@ -914,6 +921,7 @@ int unifyfs_fid_create_file(const char* path)
 {
     /* check that pathname is within bounds */
     size_t pathlen = strlen(path) + 1;
+    int rc;
```

Comment:
style: let's either declare rc this at the very to of the function followed by a blank line, or move it down into the area where it is used like ```int rc = seg_tree_init(&meta->seg_tree);```